packages/x-components/src/x-modules/tagging/service/__tests__/utils/mock-storage.service.ts

```diff
@@ -0,0 +1,37 @@
+import { StorageService } from '@empathyco/x-storage-service';
+
+/**
+ * Mocked version of the real Storage Service for testing.
+ *
+ * @internal
+ */
+export class MockedStorageService extends StorageService {
+  public getSessionIdSpy = jest.fn();
+  public setSessionIdSpy = jest.fn();
+  public removeSessionIdSpy = jest.fn();
+
+  public constructor(protected _prefix: string) {
+    super(undefined, _prefix);
+  }
+
+  getPrefix(): string {
+    return this._prefix;
+  }
+
+  injectGetResponse(key: string): void {
```

Comment:
This name is a little bit weird as I needed to look to the code to see the real intention, what about `mockSessionIdValue`?